src/vega.schema.ts

```diff
@@ -180,3 +180,13 @@ export interface VgImputeTransform {
   method?: 'value' | 'median' | 'max' | 'min' | 'mean';
   value?: any;
 }
+
+export interface VgLegendBase {
+  entryPadding?: number;
```

Comment:
1) Move this to legend.ts?  (The concept of LegendBase doesn't exist in Vega.)

2) Missing description comment for all properties.  (When you review your PR, you should inspect the schema too. If you did that, you would have notice missing "description" from the schema file.)